### PR TITLE
Feat/68 create end to end testing setup

### PR DESCRIPTION
**Pull request by decrek**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,6 +22,8 @@
     "build:server": "netlify-lambda build src/server/functions/",
     "predata": "rimraf src/client/static/data/ src/client/static/_redirects",
     "data": "dato dump",
+    "e2e": "NODE_ENV=testing run-s prebuild build:client && cypress run --config fileServerFolder=dist/client",
```
Comment (phortuin) [inline package.json]: I'd say `test` as environment name (development, test, acceptance, production)

File: package.json [hunk 2]
```diff
@@ -22,6 +22,8 @@
...
+    "e2e-dev": "cypress open --config baseUrl=http://localhost:5326",
```
Comment (phortuin) [inline package.json]: Should the task be "e2e:dev" for consistency? Not sure though

File: src/client/plugins/directives.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import Vue from 'vue'
+
+Vue.directive('test', {
+  inserted(el, binding) {
+    if(process.env.NODE_ENV === 'testing' || process.env.NODE_ENV === 'development') {
```
Comment (phortuin) [inline src/client/plugins/directives.js]: `test` here as well

File: test/cypress/plugins/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+// ***********************************************************
```
Comment (phortuin) [inline test/cypress/plugins/index.js]: Get rid of the comments

File: test/cypress/support/commands.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```
Comment (phortuin) [inline test/cypress/support/commands.js]: Can you leave out this file?

File: test/cypress/support/index.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```
Comment (phortuin) [inline test/cypress/support/index.js]: Can you leave out this as well?

**Review by phortuin — DISMISSED** [review]
I haven't looked specifically at the Vue implementation because I have no experience with Vue nor testing with Vue :)

**Review by jbmoelker — DISMISSED** [review]

**Review by jbmoelker — APPROVED** [review]

**Pull request merged**